### DISCUSSION TITLE
Error loading Python DLL 'C:\Users\xx\script\script.exe' LoadingLibrary: PyInstaller: FormatMessageW failed.

### DISCUSSION THREAD

**Discussion by Clancy-wu** [pyinstaller/pyinstaller#6499] [PyInstaller] [answered]
Hi, I package my script in windows10 via pyinstaller, and it works well in my computer. When I use this generated 'dict' dictionary in windows7 64 bit Professional Edition, the script.exe continue to flash back and nothing happened. The error information is follow:
"Error loading Python DLL "C:\Users\xx\Desktop\script\python39.dll.
No error messages generated.
LoadLibrary: PyInstaller: FormatMessageW failed."
I ensure that all the dll files is in my script dictionary, including python39.dll. 
My tries include: 1) install VC++2015 in window7 64 bit; 2) copy all the dll files in 'dict' dictionary to 'C:\Windows\system32', including python39.dll. However, the script.exe still continue to flash back. 
Could you tell me how to deal with this error? What should I do to run my exe program in window7? Because of the window7 computer belongs to the hospital, I can't update system to window10.
Looking forward to your reply. Much thanks.

**Comment by rokm** [comment, marked as answer]
You need to use older python version (3.8). Python 3.9 and newer cannot be used on Windows 7 and earlier.

**Reply by Clancy-wu** [reply]
Much appreciated !!!! Python38 works well in window7.
Thanks a lot for your quick reply.
Best wish.
